Pedersen_N02/AmrWind/abl.inp

```diff
@@ -77,3 +77,81 @@ diffusion.mg_rtol = 1.0e-6
 diffusion.mg_atol = 1.0e-12
 temperature_diffusion.mg_rtol = 1.0e-10
 temperature_diffusion.mg_atol = 1.0e-13
+
+# Activate data probe sampling
+incflo.post_processing = sampling
+
+# Frequency of output for the data
+sampling.output_frequency = 5
```

Comment:
Don't forget to change sampling to every iteration
```
sampling.output_frequency = 5
```

Reply:
Thanks. I ran out an input file with a sampling frequency of 100 upto 80000s and then bumped the sampling frequency to 1. How do we want to handle this in the repo? Do we want to upload both input files?

Reply:
Great, thanks @gantech.  So far I've just been uploading the initial input file, with notes in the file to "uncomment this section" when sampling for spectra.  If you want, we can upload both input files, so there's no ambiguity about how things were sampled.

Reply:
Thanks. I'll follow the same logic then.